tests/cancun/eip4844_blobs/test_blob_txs.py

```diff
@@ -925,23 +1056,34 @@ def test_invalid_blob_hash_versioning_multiple_txs(
 
 
 @pytest.mark.parametrize(
-    "destination_account,tx_error", [(None, "no_contract_creating_blob_txs")], ids=[""]
-)
-@pytest.mark.skip(reason="Unable to fill due to invalid field in transaction")
+    "tx_gas", [500_000], ids=[""]
+)  # Increase gas to account for contract creation
 @pytest.mark.valid_from("Cancun")
 def test_invalid_blob_tx_contract_creation(
```

Comment:
Re. chat with geth could we maybe call this `test_decode_blob_tx_contract_creation`?

With a future plan where we highlight the test type in the name:
1) `test_invalid...`, tests for expected invalidity in a block. `evm blocktest` response must be an error and match that of hive.
2)` test_decode...`, tests for the rejection of an invalid block due to a decoding error. These must pass `evm blocktest` as the block is not decoded successfully, although the logs checking the latter must match an expected error in hive.

Reply:
I'm not sure if for this specific case we could make a `decode` test, because other clients might choose to not modify their decoders for type-3 transactions and simply add a check that the value is not null after decoding.

I think both ways of implementing this check are valid.

If we create a `test_decode` fixture format, it could be a subset of invalid block tests, and we should be able to produce a blockchain test with such tests, but in this very specific case, I think should still be a blockchain test only for the reason I mentioned above.

Reply:
> I'm not sure if for this specific case we could make a decode test, because other clients might choose to not modify their decoders for type-3 transactions and simply add a check that the value is not null after decoding.

Totally agree with this, both could be considered valid